html/browsers/history/the-location-interface/location-pathname-setter-question-mark.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<title>Set location.pathname to ?</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id=log></div>
+<iframe src=/></iframe>
```

Comment:
Not sure it's great to invoke a directory listing in this test... Could it use `/common/blank.html` instead?